examples/source_file_geojson_raster.html

```diff
@@ -52,39 +52,70 @@
+                .then(function _(geojson) {
+                    return itowns.GeoJsonParser.parse(geojson, optionsGeoJsonParser);
                 }).then(function _(parsedData) {
                     var ariegeSource = new itowns.FileSource({
                         parsedData,
                     });
 
-                    var ariegeLayer = new itowns.ColorLayer('ariege', {
-                        name: 'ariege',
-                        transparent: true,
-                        style: {
-                            fill: {
-                              color: 'orange',
-                              opacity: 0.5,
-                            },
-                            stroke: {
-                                color:'white',
+                    return itowns.Fetcher.texture('images/cross.png').then(texture => {
+                        var ariegeLayer = new itowns.ColorLayer('ariege', {
+                            name: 'ariege',
+                            transparent: true,
+                            style: {
+                                fill: {
+                                  color: 'orange',
+                                  opacity: 0.5,
+                                },
+                                stroke: {
+                                    color:'white',
+                                },
                             },
-                        },
-                        source: ariegeSource,
+                            source: ariegeSource,
+                        });
+
+                        return view.addLayer(ariegeLayer).then(console.log);
+                    });
+                }));
+
+            promises.push(itowns.Fetcher.json('https://raw.githubusercontent.com/gregoiredavid/france-geojson/master/departements/66-pyrenees-orientales/departement-66-pyrenees-orientales.geojson')
+                .then(function _(geojson) {
+                    return itowns.GeoJsonParser.parse(geojson, optionsGeoJsonParser);
+                }).then(function _(parsedData) {
+                    var pyoSource = new itowns.FileSource({
+                        parsedData,
                     });
 
-                    return view.addLayer(ariegeLayer);
+                    return itowns.Fetcher.texture('images/cross.png').then(texture => {
+                        var ariegeLayer = new itowns.ColorLayer('pyrenees-orientales', {
+                            name: 'pyrenees-orientales',
+                            transparent: true,
+                            style: {
+                                fill: {
+                                  pattern: texture.image,
+                                  opacity: 0.8,
+                                },
+                                stroke: {
+                                    color:'IndianRed',
```

Comment:
How is this affecting the visual ? Is it a fallback in case the image isn't available ?

Reply:
No, it's to color the stroke.

Reply:
I can see that in the screenshot yes, sorry.